mod-audit-server/src/main/java/org/folio/builder/description/RequestDescriptionBuilder.java

```diff
@@ -154,6 +155,24 @@ public String buildReorderedDescription(JsonObject reordered) {
       .trim();
   }
 
+  public String buildExpiredDescription(JsonObject original, JsonObject updated) {
```

Comment:
The similar functionality has already implemented in the RequestStatusChangedDescriptionBuilder. Is it possible to re-use this description builder here to prevent code duplication?